apps/website/src/components/LiveExample.astro

```diff
@@ -1,7 +1,7 @@
 ---
 import fs from 'node:fs';
 import OpenInCodesandbox from './OpenInCodesandbox.astro';
-import Code from 'astro/components/Code.astro';
+import { Code } from 'astro/components';
```

Comment:
Not sure, but do the code examples from ThemeProvider use a slightly different viewer than other pages? Because I noticed that the `.expressive-code` class was only on the code examples from the ThemeProvider docs page.

Reply:
`expressive-code` is used for static examples, `Code` is used for live examples (that have a preview)

Reply:
compare the tabs page with the [deployed version](https://dev-itwinui.bentley.com/docs/tabs) to see the difference more clearly